proto/lib/petlib/ec_obj.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package syft.lib.petlib;
+
+message EC2_obj {
```

Comment:
This is the EcPt right? Can we just name it `EcPt`?

Reply:
I have named it `EC2_obj` because I used it for both, `EcPt` and `EcGroup`, I could change it to something else though